### PR TITLE
Add BHNT 67 - Hacking Memories

### PR DESCRIPTION
**Pull request by ligi**
I think I have most presentations - still missing the Hack of the Month - anyone has infos about this one?

cc @k-nut @chriseth @MichaelKreil @jpeinelt @sebastian-guent\
her @stefanhoth @benjaminwand @Wikinaut @lucasrangit

### CODE REVIEW

**Comment by Wikinaut** [comment]
pls. chg my Url from wikinaut.de to https://github.com/Wikinaut

**Comment by jpeinelt** [comment]
Can you also change my url from harpodell.de to https://github.com/jpeinelt

**Comment by ligi** [comment]
@Wikinaut @jpeinelt changes done 
@chriseth you are most welcome

File: _posts/2019-01-29-no67-hacking-memories.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: Berlin Hack & Tell \#67 - Hacking Memories
+date: 2019-01-29
+time: '18:30'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/258278925/
+---
+
+* [Minimal Presentation Software for the CLI](https://github.com/jpeinelt/skit) by [Julius Peinelt](https://github.com/jpeinelt)
+* [coline JavaScript command line framework](https://github.com/sebastian-guenther/coline) by [Sebastian](https://sebastianguenther.org)
+* [Shepard tones in Sonic Pi](https://github.com/benjaminwand/Sonic-Pi-Shepard-tones-function)
```
Comment (lucasrangit) [inline _posts/2019-01-29-no67-hacking-memories.md]: add "by [Benjamin Wand](https://github.com/benjaminwand)"

and swap position with previous talk. this was second.

File: _posts/2019-01-29-no67-hacking-memories.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* [Archiving digital data on paper](https://hydra.wikinaut.de/bhntwiki/index.php/BHNT67/Paperback) by [wikinaut](https://github.com/wikinaut)
```
Comment (lucasrangit) [inline _posts/2019-01-29-no67-hacking-memories.md]: Move paperback to last position.

**Comment by lucasrangit** [comment]
> I think I have most presentations - still missing the Hack of the Month - anyone has infos about this one?

```
* [Aposynthese](https://github.com/momonala/aposynthese) by [Mohit Nalavadi](https://momonala.github.io/) - **Hack of the month**
```

**Comment by ligi** [comment]
@lucasrangit thanks - changes applied

File: _posts/2019-01-29-no67-hacking-memories.md (hunk 2 above)
Comment (lucasrangit) [reply _posts/2019-01-29-no67-hacking-memories.md]: Looks like this line got duplicated.

**Comment by jpeinelt** [comment]
@ligi thanks!

**Pull request merged**